app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class TaskServiceTest {
+
+    private TaskService taskService;
+    private static final String TASK_TITLE = "test";
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+
+        Task task = new Task();
+        task.setTitle(TASK_TITLE);
+
+        taskService.createTask(task);
+    }
+
+    @Test
+    void getTasksList(){
+        List<Task> tasks = taskService.getTasks();
+        assertThat(tasks).hasSize(1);
+
+        Task task = tasks.get(0);
+        assertThat(task.getTitle()).isEqualTo(TASK_TITLE);
```

Comment:
```suggestion
        List<Task> tasks = taskService.getTasks();
        Task task = tasks.get(0);
        
        assertThat(tasks).hasSize(1);
        assertThat(task.getTitle()).isEqualTo(TASK_TITLE);
```

이미 0번 째의 task를 확인하고 있는데 사이즈를 검사하지 않아도 될 것 같긴 하지만, 코드와 단정문을 이렇게 정리해봐도 좋을 것 같아요

Reply:
코드와 단정문을 따로 정리하는 편이 좋나요?

Reply:
무조건 그런 것은 아닙니다 ㅎㅎ 위의 상황에서는 정리하는게 좋아보였어요. 왜냐하면 테스트하는 대상이 같고, 내용도 같다고 생각했기 때문입니다